### PR TITLE
Simplify step to add Cyberbotics APT-repository

### PR DESCRIPTION
**Pull request by daniel-cyberbotics**
:+1:

### CODE REVIEW

File: guide/installation-procedure.md [hunk 1]
```diff
@@ -47,11 +47,10 @@ The advantage of this solution is that Webots will be updated with the system
 updates. This installation requires the `root` privileges.
 
 First of all, you may want to configure your APT package manager by adding the Cyberbotics repository.
-You can easily add it from the `Software and Updates` application.
-In the `Other Software` tab, click on the `Add...` button and copy the following line:
+Simply execute the following line:
```
Comment (stefaniapedrazzi) [inline guide/installation-procedure.md]: Note that not all Webots users that work in Ubuntu are familiar with the terminal commands.
I'm not completely sure that it is good to remove the graphical procedure to add the Cyberbotics repository even if the terminal command is much simpler.

(I copied the comment from the other PR)

**Comment by stefaniapedrazzi** [comment]
Personally I would rather give both option instead of removing information.

**Comment by daniel-cyberbotics** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**